privacyidea/static/components/token/views/token.enrolled.push.html

```diff
@@ -1,5 +1,6 @@
 <div class="row">
-    <div class="col-sm-6" ng-show="enrolledToken.rollout_state === 'enrolled'">
+    <div ng-show="enrolledToken.rollout_state === 'enrolled'"
+         ng-class="$state.includes('token.wizard') == true ? 'col-sm-12' : 'col-sm-6'">>
```

Comment:
There is a typo at the end of the line. It must be ">" not ">>".

Reply:
Thank's, changed now.